airflow-core/docs/administration-and-deployment/dag-bundles.rst

```diff
@@ -83,6 +83,27 @@ For example, adding multiple dag bundles to your ``airflow.cfg`` file:
     The whitespace, particularly on the last line, is important so a multi-line value works properly. More details can be found in the
     the `configparser docs <https://docs.python.org/3/library/configparser.html#supported-ini-file-structure>`_.
 
+If you want a view url different from the default provided by the dag bundle, you can change the url in the kwargs of the dag bundle configuration.
+For example, if you want to use a custom URL for the git dag bundle:
+
+.. code-block:: ini
+
+    [dag_processor]
+    dag_bundle_config_list = [
+        {
+          "name": "my_git_repo",
+          "classpath": "airflow.dag_processing.bundles.git.GitDagBundle",
+          "kwargs": {
+            "tracking_ref": "main",
+            "git_conn_id": "my_git_conn",
+            "view_url_template": "https://my.custom.git.repo/view/{subdir}",
+          }
+        }
+      ]
+
+Above, the ``view_url_template`` is set to a custom URL that will be used to view the Dags in the ``my_git_repo`` bundle. The ``{subdir}`` placeholder will be replaced
+with the ``subdir`` attribute of the bundle. When you specify a custom URL, it overrides the default URL provided by the dag bundle.
```

Comment:
What placeholders can the user use in the template? This should be listed/described in the documentation.